tests/ansible_collections/roles/update-system/tasks/main.yml

```diff
@@ -1,2 +1,5 @@
 ---
+# We do not update Oracle Linux 8.4
 - include: update-system.yml
+  when:
+    - ansible_facts['distribution_version'] != "8.4" or ansible_facts['distribution'] != "OracleLinux"
```

Comment:
!(ansible_facts['distribution_version'] == "8.4" and ansible_facts['distribution'] == "OracleLinux") seems more readable

Reply:
@kokesak, help me understand how does the system update using this ansible role work on CentOS Linux 8.4. Is the distro image tweaked before this role is executed so that its repos point to CentOS Linux 8.4 Vault meaning that the system gets updated to the latest 8.4 packages? If so, please update the comment three lines above to:
```
# We can't update Oracle Linux 8.4 to the latest 8.4 packages because there're no public Oracle Linux 8.4 repositories available.
# The CentOS Linux 8.4 can be updated using this role because the repositories on the system image has been tweaked to point to the CentOS Linux 8.4 repositories in the Vault.
```

Reply:
@bocekm yes the distro is tweaked that only baseos, appstream and extras repos are enabled and poiting to the 8.4 url
(https://github.com/oamg/convert2rhel/pull/458/files#diff-34df60c049b010fc09dfd65d350825b2eafce0b02bd8fa487dddb730d0185193)

I'll update the comment accordingly